test/test-433-before-001.js

```diff
@@ -0,0 +1,53 @@
+var net = require('net');
```

Comment:
Looks like it's unused in test-irc.js, too. :) I'll remove it from both.

On Sun, 11 Jan 2015 01:16:26 -0800
Justin Gallardo <notifications@github.com> wrote:

> > @@ -0,0 +1,53 @@
> > +var net = require('net');
> 
> Unused require
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/martynsmith/node-irc/pull/294/files#r22765336